README.md

```diff
@@ -168,47 +168,47 @@ Binary is available for Linux, Windows and Mac OS (amd64 and arm64). Download th
 
 Linux:
 ```
-curl -sSLO https://github.com/rbalman/cfn-compose/releases/download/v0.0.1-beta/cfn-compose-v0.0.1-beta-linux-amd64.tar.gz
+curl -sSLO https://github.com/rbalman/cfn-compose/releases/download/v0.0.1-beta/cfnc-v0.0.1-beta-linux-amd64.tar.gz
```

Comment:
Let's update version to 0.0.2-beta for now. We will have to make it dynamic in our future releases.
```suggestion
curl -sSLO https://github.com/rbalman/cfn-compose/releases/download/v0.0.1-beta/cfnc-v0.0.2-beta-linux-amd64.tar.gz
```

Reply:
Changes has been applied, Thanks.